cms/envs/common.py

```diff
@@ -472,7 +473,7 @@
 
 PIPELINE_ENABLED = True
 
-STATICFILES_STORAGE = 'openedx.core.storage.ProductionStorage'
+STATICFILES_STORAGE = 'openedx.core.djangoapps.theming.storage.DevelopmentStorage'
```

Comment:
By defaulting to `DevelopmentStorage`, we ensure that users employing the `dev.py` and related environments are not running production storage.